### DISCUSSION TITLE
Tone down the reflections on the shiny textures?

### DISCUSSION THREAD

**Discussion by MrDowntempo** [sultim-t/xash-rt#153] [Ideas]
I love the working mirrors in bathrooms. And its neat to see perfect reflections in Barney's helm and the magnum, but those are... well intense, and look more fake than the originals, just because they're so perfectly reflective. I'd love to be able to either reduce the amount the reflections contribute to the texture, or just make them more diffuse.

**Comment by sultim-t** [comment]
The texture properties are defined in the `rt/data/textures.json`. Just find the revolver textures and set `isMirror` property to `false`

**Reply by MrDowntempo** [reply]
Oh, no, I don't want to turn it OFF. I want to tone it down.

**Reply by 0x4E69676874466F78** [reply]
@MrDowntempo isMirror is a hack that gives perfect reflections, if you turn it off you get "metallicDefault" and "roughnessDefault" parameters working.
`"metallicDefault":1,"roughnessDefault":0` - is real mirror but it will have delays and not work perfectly. If you set roughnessDefault to 0.1, it can be noisy on some surfaces. Experiment with this.

**Reply by MrDowntempo** [reply]
Thanks, I'll tinker around with it.
